CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## v2.0.0 (2024-10-02)
+
+- Facets changed from function to argument during execution
+- Count changed from function to argument during execution
+- Added support for request options: group_by, group_by_return_type, sort, return_explain_metadata, scoring_strategy
+- Updated documentation and README
+- Updated schema files
```

Comment:
```suggestion
- Updated schema files (version 1.47.6)
```

Reply:
Also, be sure to mention your other schema-related improvements:
- Added automatic attribute service type determination based on attribute name
- Added attribute schema exploration methods